addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2130,7 +2144,36 @@ const Wysiwyg = Widget.extend({
         this.setValue(value);
         this.odooEditor.historyReset();
         this.ptp.notifyAllClients('ptp_join');
+    },
+    _setONotEditable: function (element) {
+        const nodes = (element && element.querySelectorAll && element.querySelectorAll('.o_not_editable:not([contenteditable=false])')) || [];
+        for (const node of nodes) {
+            node.setAttribute('contenteditable', false);
+            let observer = this._oNotEditableObservers.get(node);
```

Comment:
Here are the occurences of your new map:
![image](https://user-images.githubusercontent.com/10338094/170280519-ca8fa1a2-1585-4861-8b07-314bdb20d6b7.png)

Creation, loop though, get and delete... where do you actually put something in that map? I must be dumb and missing something obvious :grimacing: